bin/ruby-build

```diff
@@ -381,10 +381,12 @@ fetch_tarball() {
     download_tarball "$package_url" "$package_filename" "$checksum"
   fi
 
-  mkdir "$package_name"
+  { if tar $tar_args "$package_filename"; then
+      if [ ! -d "$package_name" ]; then
+        extracted_dir=$(find_extracted_directory)
```

Comment:
Needs `"` around `$(...)`

Reply:
AFAIK this is correct when using assignment: https://unix.stackexchange.com/questions/118433/quoting-within-command-substitution-in-bash
I also used ShellCheck to make sure I didn't introduce any new warning.

But I'll change, it works fine with the `"` too and it's more consistent.
The above `tarball_filename` also doesn't use `"`, I'll add them.